client/src/modules/vouchers/complex-voucher.html

```diff
@@ -20,6 +20,13 @@
               </a>
             </li>
 
+            
+            <li role="menuitem">
+              <a href data-action="patient-fled" ng-click="ComplexVoucherCtrl.openPatientFledModal()">
```

Comment:
Do you use this "data-action" attribute?

Reply:
It's will be important for the test